engine/src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -13,6 +13,8 @@ object TerminalLayout {
   val Converter = new ThrowableProxyConverter
   Converter.start()
 
+  private val dateTimeFormatter = DateTimeFormatter.ofPattern("uuuu-MM-dd HH:mm:ss,SS")
```

Comment:
Also for anyone else wondering about `uuuu` - it's a fancy way of getting around "year 0" bugs and handles negative years better. `u` seems like a better general-purpose date year.